doc/cephfs/fs-volumes.rst

```diff
@@ -152,6 +152,20 @@ Fetch the absolute path of a subvolume using::
 
     $ ceph fs subvolume getpath <vol_name> <subvol_name> [--group_name <subvol_group_name>]
 
+Fetch the metadata of a subvolume using::
+
+    $ ceph fs subvolume info <vol_name> <subvol_name> [--group_name <subvol_group_name>]
+
+The output format is json and contains fields as follows.
+
+* bytes_pcent: quota used in percentage if quota is set, else displays "no-limit"
+* bytes_quota: quota size in bytes if quota is set, else displays "no-limit"
+* bytes_used: subvolume size in bytes
```

Comment:
Change it to `current used size of the subvolume` to make it similar with previous mentioning.

Reply:
Done